FV3/compile

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -eu
+
+echo
+
+if [[ ! -f conf/configure.fv3 ]]; then
+    echo "File conf/configure.fv3 not found!"
+    echo "Running ./configure .... "
+    ./configure
+fi
+
+if [[ -f conf/modules.fv3 ]]; then
+    echo -n "Loading modules .... "
+    source conf/modules.fv3
+    echo "done."
+    echo
+fi
+
+set -x
+
+echo "Running make" "$@"
+
+make clean
+make -j 8 DEBUG=Y "$@" 2>&1 | tee compile_64bit_debug.out
+mv fv3.exe fv3_64bit_debug.exe
+
+make clean
+make -j 8 "$@" 2>&1 | tee compile_64bit.out
+mv fv3.exe fv3_64bit.exe
+
+make clean
+make -j 8 OPENMP= "$@" 2>&1 | tee compile_64bit_noomp.out
+mv fv3.exe fv3_64bit_noomp.exe
+
+#make clean
+#make -j 8 DEBUG=Y 32BIT=Y "$@" 2>&1 | tee compile_32bit_debug.out
+#mv fv3.exe fv3_32bit_debug.exe
+
+#make clean
+#make -j 8 32BIT=Y "$@" 2>&1 | tee compile_32bit.out
+#mv fv3.exe fv3_32bit.exe
```

Comment:
Did you want to have this in a flag condition?

Reply:
Single precision isn't working correctly yet. This is a TODO, so to speak.